Makefile

```diff
@@ -9,6 +9,14 @@ build:
 tools:
 	GO_BUILD_FLAGS="${GO_BUILD_FLAGS} -v" ./scripts/build_tools.sh
 
+TEMP_TEST_ANALYZER_DIR=/tmp/etcd-test-analyzer
+TEST_ANALYZER_BIN=${PWD}
+bin/etcd-test-analyzer: $(TEMP_TEST_ANALYZER_DIR)/*
+	cd ${TEMP_TEST_ANALYZER_DIR}/; go build -v -o ${TEST_ANALYZER_BIN}/bin/etcd-test-analyzer; ${TEST_ANALYZER_BIN}/bin/etcd-test-analyzer -h
```

Comment:
The project supports use `make`. It can be

```
make -C ${TEMP_TEST_ANALYZER_DIR} build
mkdir -p ./bin
install ${TEMP_TEST_ANALYZER_DIR}/bin/etcd-test-analyzer ./bin/
./bin/etcd-test-analyzer -h
```

Reply:
Nice. I would like to follow that project Makefile build target.